lb.tf

```diff
@@ -84,11 +84,39 @@ resource "aws_lb_listener" "external-https" {
   }
 }
 
+resource "aws_lb_listener_rule" "deny_post_to_paths" {
+  count        = var.enable_external_lb ? 1 : 0
+  listener_arn = "${aws_lb_listener.front_end.arn}"
+  priority     = 10
+
+  action {
+    type = "fixed-response"
+
+    fixed_response {
+      content_type = "text/plain"
+      message_body = "Forbidden"
+      status_code  = "403"
+    }
+  }
+
+  condition {
+    path_pattern {
+      values = var.external_lb_paths_deny_on_post
+    }
+  }
+
+  condition {
+    http_request_method {
+      values = ["POST"]
```

Comment:
`values = var.external_lb_paths_deny_methods`

Reply:
i thought about that - it becomes a bit convoluted as all these conditions are `AND` - for instance - for metabase we do not want to block GET requests on the same uri